syntax = "proto3";

message TopicMessage{
  string object_key = 1;
  int64 user_id = 2;
  string user_email = 3;
}